src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -145,5 +156,15 @@ export default {
 			}
 		}
 	},
+	mounted() {
+		const { version } = this.apollo.readFragment({
```

Comment:
The experimentAssignment query will need to be called during prefetch for this to work